src/data/reducers/grades.js

```diff
@@ -35,8 +35,8 @@ const initialState = {
   nextPage: null,
   showSpinner: true,
   bulkManagement: {},
-  totalUsersCount: null,
-  filteredUsersCount: null,
+  totalUsersCount: 0,
```

Comment:
Without this change, the gradebook displays "Showing of total learners" as it is loading. I thought this was more presentable. I considered hiding the text all together but opted for this for now as it's easier.

Reply:
Hiding the text altogether likely wouldn't be terrifically difficult; it just amounts to adding a strategicly placed `this.props.totalUsersCount ? <jsx> : null` to the Gradebook's `render` method (whether or not we ensure its value is numeric by setting a default value here). I'd prefer that to flickering a lie that there are no learners in the course.

Reply:
That makes sense. I have added that.

I originally had mistakenly done `{this.props.totalUsersCount && <jsx>}`, which just displayed a very unhelpful `0`. Now I realize that the left hand sign of the && operator should be true or false, not a truthy or falsy value.